docs/source/quicktour.mdx

```diff
@@ -10,26 +10,41 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
+# Quicktour
 
-Let's have a look at 🤗 PEFT's main features and learn how to set up a `PeftModel` and train it with 🤗 Accelerate's DeepSpeed integration and use it for inference.
+🤗 PEFT contains parameter-efficient fine-tuning methods for training large pretrained models. The traditional paradigm is to fully train a model for each downstream task, but this is becoming exceedingly costly and impractical because of the enormous number of parameters in models today. Instead, it is more efficient to train a smaller number of prompt parameters or use a reparametrization method like low-rank adaptation (LoRA) to reduce the number of trainable parameters. 
 
-## Main use
+This quicktour will show you 🤗 PEFT's main features and help you train large pretrained models that would typically be inaccessible on consumer devices. You'll see how to train the 1.2B parameter [`bigscience/mt0-large`](https://huggingface.co/bigscience/mt0-large) model with LoRA to generate a classification label and use it for inference, and also how to train a 🤗 PEFT model with 🤗 Accelerate's DeepSpeed integration on distributed setups.
 
-To use 🤗 PEFT in your script:
+## PeftConfig
 
-1. Each PEFT method is defined by a `PeftConfig` object.
+Each 🤗 PEFT method is defined by a [`PeftConfig`] class that stores all the important parameters for configuring the model. 
 
-Create a `PeftConfig` object corresponding to your PEFT method (see the [Configuration](package_reference/config) reference for more details) and [`TaskType`], the type of task you're training your model for.
-This example trains the [`bigscience/mt0-large`](https://huggingface.co/bigscience/mt0-large) model with the Low-Rank Adaptation of Large Language Models (LoRA) method. Load the `LoRAConfig`, and specify the `task_type` for sequence-to-sequence language modeling.
+Because you're going to use LoRA, you'll need to load and create a [`LoraConfig`] class. Within `LoraConfig`, specify the following parameters:
+
+- the `task_type`, or sequence-to-sequence language modeling in this case
+- `inference_mode`, whether you're using the model for inference or not
+- `r`, the dimension of the low-rank matrices
+- `lora_alpha`, the scaling factor for the low-rank matrices
+- `lora_dropout`, the dropout probability of the LoRA layers
 
 ```python
 from peft import LoraConfig, TaskType
 
 peft_config = LoraConfig(task_type=TaskType.SEQ_2_SEQ_LM, inference_mode=False, r=8, lora_alpha=32, lora_dropout=0.1)
 ```
 
-2. Load the base model you want to fine-tune.
+<Tip>
+
+💡 See the [`LoraConfig`] reference for more details about other parameters you can adjust.
+
+</Tip>
+
+## PeftModel
+
+A [`PeftModel`] is created by the [`get_peft_model`] function. It takes a base model - which you can load from the 🤗 Transformers library - and the [`PeftConfig`] containing the instructions for how to configure a model for a specific 🤗 PEFT method, and it returns a [`PeftModel`]. 
+
+Start by loading the base model you want to finetune.
```

Comment:
Earlier in the doc you use "fine-tune". I don't mind either spellings, but it's probably best to pick one for the entire document :)